app/views/RoomView/List/components/index.ts

```diff
@@ -1,3 +0,0 @@
-export * from './NavBottomFAB';
```

Comment:
My remove barrel files PR didn't have the intention to remove every single file.
Only the ones mentioned on the description, which is quite enough 😬